code/style.css

```diff
@@ -0,0 +1,144 @@
+
+body{
+  padding: 0;
+  margin: 0;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+header {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+
+header img{
+  width: 100%;
+  height: 100vh;
+  background-size: cover;
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  filter: brightness(70%);
+}
+
+header h1{
+  text-align: center;
+  position: absolute;
+  font-family: 'Pacifico', cursive;
+  color: #F3F0DD;
+  text-shadow: 0px 0px 15px black;
+  font-size: 5rem;
+}
+
+#orderMenu {
```

Comment:
You seem to style a lot by ids and selectors - please, change that to classes only. Styling by ids and selectors is considered as bad practice because styles added by ids have higher importance than by class, and styles added by selectors have lower importance than by class